packages/documentation/src/content/docs/overview/concepts/accounts-transfers-liquidity.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Accounts, transfers, and liquidity
+---
+
+import { LinkOut } from '@interledger/docs-design-system'
+
+import { CodeBlock } from '@interledger/docs-design-system'
+
+## Accounts
+
+Rafiki uses a combination of liquidity (credit) and settlement (debit) accounts to perform <LinkOut href="https://en.wikipedia.org/wiki/Double-entry_bookkeeping">double-entry accounting</LinkOut>.
+
+In this context, accounts in Rafiki are specifically the accounts that peers hold with one another. These are not customer accounts or accounts on any underlying ledgers. This distinction is crucial for understanding how Rafiki handles transactions and settlements.
+
+### Liquidity accounts
+
+A liquidity account holds a non-negative balance, with Rafiki ensuring that total debits do not exceed total credits.
+
+There is one liquidity account for each of the following resources:
+
+- Asset
+- Peer
+- Wallet address (for SPSP/Web Monetization receiving)
+- Incoming payment
+- Outgoing payment
+
+### Settlement accounts
+
+A settlement account holds a non-positive balance, with Rafiki ensuring that total credits do not exceed total debits. A settlement account represents the total funds an Account Servicing Entity has deposited into Rafiki.
+
+There is one settlement account for each asset.
+
+## Assets
+
+An asset in Rafiki represents an item of value that can be transferred via the Interledger Protocol. Since the Interledger Protocol aims to create an internet of value, it allows for the transfer of any asset, not just currency. In practice, however, assets are usually denominated in a currency (fiat or branded currencies).
+
+Before peering with another account servicing entity, you must agree on the asset you will use for settlement. The Interledger packets exchanged between you and your peer will be denominated in the agreed-upon asset.
+
+Furthermore, an account servicing entity performing currency exchange must provide asset liquidity. Asset liquidity ensures there is enough value available to handle transactions in the specified assets.
+
+Assets are managed via the [Backend Admin API](/integration/services/backend-service#graphql-backend-admin-api) or the [Rafiki Admin application](/admin/admin-user-guide/). Wallet addresses are created for existing assets and are tied to those assets.
+
+### The `asset` type
```

Comment:
```suggestion
### The asset type
```

The heading looks odd like this otherwise 

![image](https://github.com/user-attachments/assets/fa57a168-9c40-488e-8e52-72a278cdf9e0)